icu4c/source/test/intltest/messageformat2test_utils.h

```diff
@@ -4,15 +4,17 @@
 #ifndef _TESTMESSAGEFORMAT2_UTILS
 #define _TESTMESSAGEFORMAT2_UTILS
 
+#if !UCONFIG_NO_FORMATTING
```

Comment:
Please make sure that before any of these config switches are checked, at least one of the public header files has been included so that the compiler has seen uconfig.h.

If in doubt, include unicode/utypes.h then check the config switches then include the rest of the headers.

Reply:
Fixed in [d744f2c](https://github.com/unicode-org/icu/pull/2931/commits/d744f2cd1ed135048b620ce10f2c2e31ae58ec3b)